static/templates/message_edit_history.handlebars

```diff
@@ -7,7 +7,7 @@
     <div class="messagebox-border">
         <div class="messagebox-content">
             <div class="message_top_line"><span class="message_time">{{ timestamp }}</span></div>
-            <div class="message_edit_content">{{{ body_to_render }}}</div>
+            <div class="message_edit_history_content">{{{ body_to_render }}}</div>
```

Comment:
I think this still isn't right.  It seems like we might want this to just be `message_content message_edit_history_content`, and mostly inherit all the styling we already have for things inside `message_content`?  

Certainly the resize CSS is a feature of the `message_edit` UI widget, which has nothing to do with how we should display a series of messages with their diffs.